src/main/scala/fs2/StreamOps.scala

```diff
@@ -40,11 +46,16 @@ trait StreamOps[+F[_],+A] extends Process1Ops[F,A] /* with TeeOps[F,A] with WyeO
   def merge[F2[_]:Async,B>:A](s2: Stream[F2,B])(implicit R: RealSupertype[A,B], S: Sub1[F,F2]): Stream[F2,B] =
     fs2.wye.merge.apply(Sub1.substStream(self), s2)
 
+  def onComplete[F2[_],B>:A](regardless: => Stream[F2,B])(implicit R: RealSupertype[A,B], S: Sub1[F,F2]): Stream[F2,B] =
+    Stream.onComplete(Sub1.substStream(self): Stream[F2,B], regardless)
+
   def onError[F2[_],B>:A](f: Throwable => Stream[F2,B])(implicit R: RealSupertype[A,B], S: Sub1[F,F2]): Stream[F2,B] =
     Stream.onError(Sub1.substStream(self): Stream[F2,B])(f)
 
   def open: Pull[F, Nothing, Handle[F,A]] = Stream.open(self)
 
+  def outputs: Pull[F,A,Unit] = Stream.outputs(self)
```

Comment:
Just call this output (no s) and have it call Pull.outputs. Delete Stream.outputs. Idea is "output this stream from within a Pull"